cabal.project.newbuild

```diff
@@ -31,3 +31,9 @@ constraints:
   regex == 1.0.1.3
 allow-newer:
   regex:template-haskell
+
+-- TODO RGS: Remove this when on Hackage
```

Comment:
Version 0.2.4.0 of s-cargot-letbind is now released on Hackage.

Reply:
Thanks! Some of `semmc-semstore-tests`'s test cases [now fail](https://github.com/GaloisInc/semmc/actions/runs/4731593877/jobs/8396718189#step:10:79) with this version of `s-cargot-letbind`:

```
     DSL specified
      Trivial formula, OpSolo:                  FAIL
        tests/semstore/ParamFormulaTests.hs:404:
        expected: ";; OpSolo (no arguments)\n((operands ())\n (in ())\n (defs ()))"
         but got: ";; OpSolo (no arguments)\n((operands ())\n  (in ())\n  (defs ()))"
        Use -p '/Trivial formula, OpSolo/' to rerun this test only.
```

But based on the changes in https://github.com/GaloisInc/s-cargot-letbind/commit/e3d95753ce548a1899d1a24d35b0aabe258c0f42, I'm guessing that some output differences are to be expected. In the spirit of that commit, I've opted to bump the `s-cargot` lower version bounds to `>= 0.1.6.0` and accept the new test output.